registry/consumer/service_event_consumer.py

```diff
@@ -125,7 +125,7 @@ def _process_service_data(self, org_id, service_id, service_metadata, tags_data,
         }
         assets = service_metadata.get("assets", {})
         mpe_address = service_metadata.get("mpe_address", "")
-        metadata_ipfs_hash = service_metadata.get("metadata_ipfs_hash", "")
+        metadata_uri = "ipfs://" + metadata_uri
```

Comment:
The metadata url is of the form from below from blocchain 
ipfs://QmXfW7gmpQUVwYseAfyGS8ygEPKugfse4jQvgCvD6u4h9J           
dont see the stripping of ipfs  => do you really need to add this "ipfs://", can you reconfirm this 
 _get_metadata_uri_from_event(self, event):
        event_data = event['data']
        service_data = eval(event_data['json_str'])
        metadata_uri = Web3.toText(service_data['metadataURI'])[7:].rstrip("\u0000")
        return metadata_uri